packages/core/src/components/Table/Body/Body.styled.tsx

```diff
@@ -1,6 +1,10 @@
-import { styled } from '@medly-components/utils';
+import { getFontStyle, styled } from '@medly-components/utils';
+import Text from '../../Text';
 
 export const TBody = styled('tbody')`
     display: block;
     min-width: fit-content;
+    ${Text.Style} {
```

Comment:
@merewelch214 I think we should add this in [`Row`](https://github.com/medly/medly-components/blob/master/packages/core/src/components/Table/Body/Row/Row.styled.tsx#L116) component.

Reply:
moved to `Row` component. thanks!